time stamp
int64 data1
int64 data2
string message
